lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +176,9 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    time.sleep(random.randrange(1, 3))
+    return requests.get(url=url, timeout=config.get_timeout(), headers=config.get_headers(),
```

Comment:
What is the purpose of `encoding` parameter of scrapper configuration? Where do you think it could be used?

Reply:
This parameter specifies encoding for the response received by the web page you request